<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="5" passed="1" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-06T15:37:46 EDT" name="Regression" finished-at="2022-09-06T15:37:47 EDT" duration-ms="81">
    <groups>
    </groups>
    <test started-at="2022-09-06T15:37:46 EDT" name="Release001" finished-at="2022-09-06T15:37:47 EDT" duration-ms="81">
      <class name="c11Practice.executionOrder.PriorityDisabledDependencyTesting">
        <test-method signature="practiceReporting()[pri:0, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47]" started-at="2022-09-06T15:37:47 EDT" name="practiceReporting" finished-at="2022-09-06T15:37:47 EDT" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- practiceReporting -->
        <test-method signature="loginWithValidCredentials()[pri:1, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47]" started-at="2022-09-06T15:37:47 EDT" name="loginWithValidCredentials" finished-at="2022-09-06T15:37:47 EDT" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[PurposelyFailing]]>
            </message>
            <full-stacktrace>
                <![CDATA[java.lang.AssertionError: PurposelyFailing
at org.testng.Assert.fail(Assert.java:97)
at c11Practice.executionOrder.PriorityDisabledDependencyTesting.loginWithValidCredentials(PriorityDisabledDependencyTesting.java:11)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidCredentials -->
        <test-method signature="logoVerification()[pri:3, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47]" started-at="2022-09-06T15:37:47 EDT" name="logoVerification" finished-at="2022-09-06T15:37:47 EDT" depends-on-methods="c11Practice.executionOrder.PriorityDisabledDependencyTesting.loginWithValidCredentials" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
                <![CDATA[Method PriorityDisabledDependencyTesting.logoVerification()[pri:3, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
                <![CDATA[java.lang.Throwable: Method PriorityDisabledDependencyTesting.logoVerification()[pri:3, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoVerification -->
        <test-method signature="logOutFromHrmApplication()[pri:4, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47]" started-at="2022-09-06T15:37:47 EDT" name="logOutFromHrmApplication" finished-at="2022-09-06T15:37:47 EDT" depends-on-methods="c11Practice.executionOrder.PriorityDisabledDependencyTesting.loginWithValidCredentials" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
                <![CDATA[Method PriorityDisabledDependencyTesting.logOutFromHrmApplication()[pri:4, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
                <![CDATA[java.lang.Throwable: Method PriorityDisabledDependencyTesting.logOutFromHrmApplication()[pri:4, instance:c11Practice.executionOrder.PriorityDisabledDependencyTesting@7fe8ea47] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:101)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOutFromHrmApplication -->
      </class> <!-- c11Practice.executionOrder.PriorityDisabledDependencyTesting -->
    </test> <!-- Release001 -->
  </suite> <!-- Regression -->
</testng-results>
